README.md

```diff
@@ -1 +1,18 @@
-# customerio-reactnative
\ No newline at end of file
+# Customer.io ReactNative SDK
+
+This is the official Customer.io SDK for ReactNative.
```

Comment:
Since you are already making improvements to the README, mind also [adding our CIO logo and optionally some badges](https://github.com/customerio/customerio-ios/blob/545148820187c04938e717fce4e166cd77a78f92/README.md#L1-L19) to the README like our native SDKs? 

For badges, perhaps [a npm](https://shields.io/category/downloads) badge for latest version available?

Reply:
Added `npm` badges and removed `codecov` as it shows `unknown` for this repo currently.